ghcide/ghcide.cabal

```diff
@@ -2,7 +2,7 @@ cabal-version:      2.4
 build-type:         Simple
 category:           Development
 name:               ghcide
-version:            1.3.0.0
+version:            1.3.0.1
```

Comment:
did you intend to bump the ghcide version?

Reply:
I thought I should bump it because I did one change in ghcide, expose one more module which is needed to calculate whole line removal